zebedee-cms/src/main/java/com/github/onsdigital/zebedee/model/approval/ApproveTask.java

```diff
@@ -280,9 +280,16 @@ private void compressZipFiles(Collection collection, CollectionReader collection
 
     public void approveCollection() throws IOException {
         // set the approved state on the collection
-        collection.description.approvalStatus = ApprovalStatus.COMPLETE;
-        collection.description.addEvent(new Event(new Date(), EventType.APPROVED, session.getEmail()));
-        collection.save();
+        try {
+            collection.getDescription().setApprovalStatus(ApprovalStatus.COMPLETE);
+            collection.getDescription().addEvent(new Event(new Date(), EventType.APPROVED, session.getEmail()));
+            collection.save();
+        } catch (Exception ex) {
```

Comment:
Minor consistency comment most places we have been keeping exception name as `e` but here it is `ex`.

Reply:
By convention it should be `ex`.